programmers2/두-개-뽑아서-더하기.js

```diff
@@ -0,0 +1,12 @@
+function solution(numbers) {
+    let sum = [];
```

Comment:
const 가 맞는 변수 키워드입니다!
```suggestion
    const sum = [];
```